tests/conftest.py

```diff
@@ -72,305 +58,35 @@ def polarion_testsuite_properties(record_testsuite_property):
         )
 
 
-def cluster_teardown(log_level="DEBUG"):
-    log.info("Destroying the test cluster")
-    destroy_cluster(config.ENV_DATA['cluster_path'], log_level)
-    log.info("Destroying the test cluster complete")
-
-
 @pytest.fixture(scope="session", autouse=True)
 def cluster(request, log_cli_level):
+    """
+    This fixture initiates deployment for both OCP and OCS clusters.
+    Specific platform deployment classes will handle the fine details
+    of action
+    """
     log.info(f"All logs located at {log_path}")
     log.info("Running OCS basic installation")
```

Comment:
This log should be probably printed from deploy_ocs method as we can skip the OCS installation now and have just OCP installation

Reply:
Done